crates/rspack_plugin_javascript/tests/fixtures/builtins/minify/test.config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "builtins": {
+        "minifyOptions": {
+            "extractComments": "/@license/"
+        },
+        "codeGeneration": {
```

Comment:
I don't think this option going to work, since your `keepComments` only work when mode is `development` the comments will be eliminated by minifier anyway.

Reply:
`keepComments` is not depend on `mode`, it is only affected by the `codeGeneration.keepComment` option, that means we can keep comments even though at `production` mode, but don't worry about bundle size, minify_file_comments will minify it.
https://github.com/web-infra-dev/rspack/blob/e764de62c21615380b6063ac3f16e74d6f9b165f/crates/rspack_plugin_javascript/src/ast/minify.rs#L176

Reply:
Maybe you misunderstand me,
I think `codeGenerationd.keepComments` should be `true` by default when the user enables the `minifyOptions.extractComments`(obviously user wants to keepComments when using `extractComments`), two options may cause users extra mental burden.